frontend/upload.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>Oklahoma Department of Human Services</title>
+
+        <!--viewport metas go here-->
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <link rel="stylesheet" href="base.css">
+        <link rel="stylesheet" href="styles.css">
+    </head>
+
+    <body>
+    	<header class="banner">
+    		<img class="logo" src="" alt="DHS logo">
+    		<a class="web_title" href="index.html">OKDHS Adult and Family Services Information Center</a>
+
+            <nav class="generalNav">
+                <ul>
+                    <li><a href="profile.html"><img src="" alt="profile">Profile</a></li>
+                    <li><a href="calendar1.html"><img src="" alt="calendar">Calendar</a></li>
+                    <li><a href=""><img src="" alt="upload">Uploads</a></li>
+                    <li><a href="help1.html"><img src="" alt="help">Help</a></li>
+                </ul>
+            </nav>
+    	</header>
+
+    	<article>
+    	   <h1>Image Uploading Center</h1>
+
+           <section class="uploaded">
+                <h2>Previous Upload</h2>
+                <table>
+                    <tr>
+                        <th>Name</th>
+                        <th>Date Uploaded</th> 
+                        <th>Type</th>
+                        <th>View</th>
+                    </tr>
+                    <tr>
+                        <td>Walmart Check</td>
+                        <td>4/13/2019</td>
+                        <td>Paystub</td>
+                        <td><a href="">open</a></td>
+                    </tr>
+                    <tr>
+                        <td>Walmart Check</td>
+                        <td>4/27/2019</td>
+                        <td>Paystub</td>
+                        <td><a href="">open</a></td>
+                    </tr>
+                    <tr>
+                        <td>Walmart Check</td>
+                        <td>5/12/2019</td>
+                        <td>Paystub</td>
+                        <td><a href="">open</a></td>
```

Comment:
Need to discuss how we want to handle this. Do we want a new page, or to have a embedded page with the image

Reply:
new tab